src/packages/core/src/server.ts

```diff
@@ -283,6 +292,19 @@ export class Server<
     }
   }
 
+  public address() {
```

Comment:
This IP copies Node's own http `createServer`'s `address()` API: https://nodejs.org/api/net.html#net_server_address

With one exception: we do _not_ support named pipes and Unix domain sockets, so we do need to support returning a string.